### ISSUE TITLE
Create a new release to include actionlint 1.6.23

### ISSUE THREAD

**Issue by jacobwoffenden** [github/super-linter#3965]
Hi 👋,

I am currently implementing Super-Linter, but it is failing with

```bash
------
.github/workflows/terraform-github.yml:36:23: undefined variable "vars". available variables are "env", "github", "inputs", "job", "matrix", "needs", "runner", "secrets", "steps", "strategy" [expression]
   |
36 |       AWS_REGION: ${{ vars.AWS_REGION }}
   |                       ^~~~~~~~~~~~~~~
------
```

This is fixed in https://github.com/rhysd/actionlint/releases/tag/v1.6.23, and has been merged into main https://github.com/github/super-linter/pull/3852

Is it possible to get an estimate on a new release of Super-Linter?

**Comment by G-man83** [comment]
We are also waiting for this 😄 I've disabled `actionlint` in our workflow in the meantime, until super-linter is updated.

**Comment by neilime** [comment]
In general, it would be good to have regular releases. super-linter is a great project, but is is quite tricky to keep using it in our workflows on a daily basis, as the linters are  frequently not up to date.

Thank you BTW for your work

**Comment by KaoutherElhamdi** [comment]
Yes I have the same issue but for SQLfluff linter
I want to upgrade to the latest released version 2.0.2 but unfortunately we are blocked with super-linter 
That will be cool if they can upgrade the version

**Comment by YElyousfi** [comment]
When will the action be updated? Overall, what is the schedule for releases?

**Comment by lindluni** [comment]
All updates have been released as `v5.0.0`
